pkg/arvo/sys/lull.hoon

```diff
@@ -1026,90 +1025,82 @@
         [%crud p=@tas q=(list tank)]                    ::  print error
-        [%flee session=~]                               ::  unwatch session
         [%flog p=flog]                                  ::  wrapped error
-        [%flow p=@tas q=(list gill:gall)]               ::  terminal config
-        [%hail ~]                                       ::  terminal refresh
         [%heft ~]                                       ::  memory report
-        [%hook ~]                                       ::  this term hung up
-        [%harm ~]                                       ::  all terms hung up
         $>(%init vane-task)                             ::  after gall ready
         [%meld ~]                                       ::  unify memory
-        [%noop ~]                                       ::  no operation
         [%pack ~]                                       ::  compact memory
-        [%talk p=tank]                                  ::
-        [%text p=tape]                                  ::
-        [%view session=~]                               ::  watch session blits
+        [%shot ses=@tas task=session-task]              ::  task for session
+        [%talk p=(list tank)]                           ::  print tanks
+        [%text p=tape]                                  ::  print tape
         $>(%trim vane-task)                             ::  trim state
         $>(%vega vane-task)                             ::  report upgrade
         [%verb ~]                                       ::  verbose mode
         [%knob tag=term level=?(%hush %soft %loud)]     ::  error verbosity
+        session-task                                    ::  for default session
+    ==                                                  ::
+  ::                                                    ::
+  +$  session-task                                      ::  session request
+    $%  [%belt p=belt]                                  ::  terminal input
+        [%blew p=blew]                                  ::  terminal config
+        [%flee ~]                                       ::  unwatch session
+        [%hail ~]                                       ::  terminal refresh
+        [%open p=dude:gall q=(list gill:gall)]          ::  setup session
+        [%shut ~]                                       ::  close session
+        [%view ~]                                       ::  watch session blits
     ==                                                  ::
   ::
   ::::                                                  ::  (1d2)
     ::
   +$  blew  [p=@ud q=@ud]                               ::  columns rows
-  +$  belt                                              ::  old belt
+  +$  belt                                              ::  client input
+    $?  bolt                                            ::  simple input
+        [%mod mod=?(%ctl %met %hyp) key=bolt]           ::  w/ modifier
```

Comment:
Are we supposed to not be able to represent eg "control alt C"?  Feels like this should be `key=belt`

Reply:
Or maybe just `mod=(set ?(%ctl %met %hyp))`

Reply:
`%hyp` here represents the simultaneous pressing of `%ctl` and `%met`, not a single "hyper key".

Reply:
Ah, okay